apps/studio/electron/preload/webview/dom.ts

```diff
@@ -65,6 +117,11 @@ export function buildLayerTree(root: HTMLElement): LayerNode | null {
 function processNode(node: HTMLElement): LayerNode {
     getOrAssignUuid(node);
 
+    const isDynamic = node.hasAttribute(EditorAttributes.DATA_ONLOOK_DYNAMIC);
+    const dynamicType: any = isDynamic
```

Comment:
It maybe be worth keeping the types in an enum somewhere instead of have it be anything. In case we want to react a certain way to a type.

Reply:
oh yeah, That skipped me. I will update that